src/xdocs/config.xml

```diff
@@ -729,6 +729,33 @@
       </p>
     </section>
 
+    <section name="Id">
+      <p>
+        Each module has a <code>id</code> property that could be used to differentiate
+        two instances of the same Check. It is helpful to make different rules in suppressions
```

Comment:
After `Check.`
`This custom module name should be unique for the entire configuration to prevent accidental overlapping.`